<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="listaDeCustos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <column>
    <property name="text">
     <string>Custo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor (R$)</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Valor R$</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="novoCustoText"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="novoValorText"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="botaoAddNovoCusto">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Renda Mensal:  R$</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="rendaMensalText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{background-color: qlineargradient(spread:pad, x1:0.512, y1:0, x2:0.518, y2:1, stop:0 rgba(254, 255, 172, 255), stop:1 rgba(255, 170, 127, 255));}
</string>
   </property>
   <widget class="QPushButton" name="botaoExcluir">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoVoltar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Adicionar Novo Custo Fixo</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="listaSetor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Setor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Porcentagem</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Porcentagem</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="novoSetor"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="novaPorcentagem"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Adicionar Novo Setor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoAddNovoSetor">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>listaDeCustos</zorder>
  <zorder>formLayoutWidget_2</zorder>
  <zorder>botaoAddNovoCusto</zorder>
  <zorder>formLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
